frontend/src/css/imports.css

```diff
@@ -5562,7 +5562,7 @@ button.close {
   position: absolute;
   top: 0;
   left: 0;
-  z-index: 1060;
+  z-index: 1090;
```

Comment:
Doesn't seem like any element is using this class, is this change for future usage? (to popover the header?)

Reply:
Yeah we use .popover somewhere else and just because I made changes to the worksheet title and CLI, I wanted to assign a higher z-index to .popover than any of the others. 

Is this a good practice? @epicfaace

Reply:
Sounds good

Reply:
Yeah, that's good.